src/openforms/js/components/admin/form_design/registrations/zgw/utils.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const getChoicesFromSchema = (enums, enumNames) => {
+  let finalChoices = {};
+  Object.keys(enums).forEach(key => {
+    finalChoices[enums[key]] = enumNames[key];
+  });
+
+  return finalChoices;
+};
+
+const getFieldErrors = (name, index, errors, field) => {
+  const errorMessages = [];
+
+  for (const [errorName, errorReason] of errors) {
+    if (errorName.startsWith(name)) {
+      const errorNameBits = errorName.split('.');
+      if (errorNameBits[2] === String(index) && errorNameBits[errorNameBits.length - 1] === field) {
+        errorMessages.push(errorReason);
+      }
+    }
+  }
+
+  return errorMessages.length > 0 ? errorMessages : null;
+};
+
+const getErrorMarkup = errorMessages => {
+  return (
+    <div className="rjsf-field__errors">
+      <div>
+        <ul className="error-detail bs-callout bs-callout-info">
+          {errorMessages.map((msg, index) => (
+            <li key={index} className="text-danger">
+              {msg}
+            </li>
+          ))}
+        </ul>
+      </div>
```

Comment:
Why the extra div without class name? Also, would it make sense to add this to the `CustomFieldTemplate` instead of calling it per custom field?

Reply:
Yes, I removed this div. The way I implemented this has to do with what the `CustomFieldTemplate` component receives for the errors, which is a `ReactElement`. So I thought it would be better to do this explicitly.